android/quest/src/main/java/org/smartregister/fhircore/quest/ui/login/AccountAuthenticator.kt

```diff
@@ -78,8 +80,12 @@ constructor(
         authToken =
           try {
             tokenAuthenticator.refreshToken(refreshToken)
-          } catch (httpException: HttpException) {
-            ""
+          } catch (ex: Exception) {
```

Comment:
@owais-vd have you tried the unknown host exception?

Reply:
yes tired and got `UnknownHostException`

Reply:
k can we update to catch both HttpException and UnknownHostException? We'd like to avoid using a generic

Reply:
oh sorry! I misunderstood and code and didn't realize this is a strategy for multicatch in kotlin, this makes sense!